web-console/1-homepage/search.md

```diff
@@ -1,50 +1,57 @@
-# Template
+# Search
 
-- This is a description of the section and where it is located/how to get to it
-- This is a description of what this section is for
+- The search bar is located at the very top of the web console homepage
+- This component enables users to enter a query and receive a list of results.
 
 
-## Use Case 1 (e.g. Browse for service)
+## Search for Service
 
-![template](img/template-01.png)
-- description of what is happening in this wireframe
+![search](img/search-01.png)
+- Deploy Image and Import YAML/JSON buttons are available next to the search bar and trigger the relevant [overlay panel actions](http://openshift.github.io/openshift-origin-design/web-console/4-patterns/overlay-panel).
 
 #### Implementation Details
-  - Note PatternFly design patterns
-  - Note icon references
+  - Import YAML/JSON link may not be necessary on this page
 
-![template](img/template-02.png)
-- description of what is happening in this wireframe
+![search](img/search-02.png)
+- Entering a search term brings up a list of the first five matches.
+  - Clicking on a match will trigger the order flow for that service on the [overlay panel](http://openshift.github.io/openshift-origin-design/web-console/4-patterns/overlay-panel).
```

Comment:
We may want to change the wording here, rather than saying service say catalog item maybe?  Since these can be either builder images or services @cshinn